.github/CODEOWNERS

```diff
@@ -1,5 +1,4 @@
-# All members of the Code for Lansing Leadership team are considered code
-# owners for all new projects. Additional individuals and teams can be added
-# on a per-project basis.
+# All members of the Code for Lansing Cityzen Maintainers team are considered
+# code owners for this project.
 
-*       @codeforlansing/leadership
+*       @codeforlansing/cityzen-maintainers
```

Comment:
We should consider adding github actions to the template that autogenerate these changes at some point in the future

Reply:
Agreed! I created codeforlansing/project-template#5 to remind us.